quantized_mesh_encoder/normals.py

```diff
@@ -44,6 +44,10 @@ def compute_vertex_normals(positions, indices):
     return normalized_vertex_normals
 
 
+def signNotZero(v):
+    return -1.0 if v < 0.0 else 1.0
```

Comment:
This needs to be vectorized.
```suggestion
def sign_not_zero(arr):
    """A variation of np.sign that coerces 0 to 1"""
    signed = np.sign(arr)
    return np.where(signed == 0, 1, signed)
```

Reply:
I committed this together with the rest o the changes requested below so it'd all work together. Thanks!